src/extensions/carbon.cr

```diff
@@ -0,0 +1,15 @@
+require "carbon"
+require "./email_previews"
+
+require "./actions/**"
+require "./pages/**"
+
+module Breeze
+  Habitat.extend do
+    setting email_previews : Carbon::EmailPreviews.class, example: "Emails::Previews"
+  end
+end
+
+Breeze::SidebarLinks.settings.links << ->(breeze : Breeze::SidebarLinks) {
+  breeze.mount(Breeze::SidebarLink, context: breeze.context, link_text: "Emails", link_to: Breeze::Emails::Index)
+}
```

Comment:
This feels weird putting component related info in to this base class, but since it's setting a Habitat config value, where else would it go?